src/downshift.js

```diff
@@ -181,9 +189,12 @@ class Downshift extends Component {
   ) => {
     otherStateToSet = pickState(otherStateToSet)
     this.internalSetState({highlightedIndex, ...otherStateToSet}, () => {
-      const node = this.getItemNodeFromIndex(this.getState().highlightedIndex)
-      const rootNode = this._rootNode
-      scrollIntoView(node, rootNode)
+      // TODO: this whole method is DOM specific...
```

Comment:
Should we just omit this logic here? This is specific to DOM and to get this to work with React Native, we'd have to force consumers to use a `ScrollView` and pass us the ref.

Reply:
Yeah, we'll probably want some different logic here for React Native